test/controllers/LogoutControllerSpec.scala

```diff
@@ -44,21 +44,25 @@ class LogoutControllerSpec extends BaseSpec {
         session(result).get(SessionKeys.token) mustBe empty
 
       }
-      "when already logged out" in {
 
-        val controller: LogoutController = inject[LogoutController]
+      "when already logged out" in new LogoutTestCase {
```

Comment:
Why the change in formatting? 

It seems inconsistent both with the style in this class, and elsewhere, where we don't have newlines between these scenario statements.

Reply:
Sorry, I don't understand the question - is it referring to new lines between the test scenarios?

Reply:
I added the newline here because I think that test declarations as similar to function declarations - with a new line before and after the definition.

Reply:
I leave it up to individuals how much white space they put in a file - within reason of course.